docs/src/pages/components/checkboxes/checkboxes.md

```diff
@@ -22,6 +22,13 @@ If you have a single option, avoid using a checkbox and use an on/off switch ins
 
 {{"demo": "pages/components/checkboxes/Checkboxes.js"}}
 
+## Indeterminate
+
+A checkbox input can only have two states: checked or unchecked. It either submits its value or doesn't.
```

Comment:
Could you check how the submitted formdata looks? Because a checkbox is an actual tri-state. Not just a visual one.

Reply:
I'm not sure to understand the question. A playground: https://codesandbox.io/s/black-rgb-hbw1f?file=/demo.js. Maybe with its clearer?
```suggestion
A checkbox input can only have two states in a form: checked or unchecked. It either submits its value or doesn't.
```

Reply:
That would be correct. It still seems you're not fully understanding that a checkbox is a tri-state though that isn't my concern at this point. What's important is that we don't communicate wrong facts.

Reply:
So with the suggested wording, we are good? :) checkbox are form data submission two-states, and visually tri-state?